### ISSUE TITLE
Security console sorting - criminal status

### ISSUE THREAD

**Issue by 00aquir** [yogstation13/yogstation#2034]
Please make it possible to sort by criminal status.

**Comment by Mrcreepssss** [comment]
i agree this would be nice, but seeing as each status has aa very different color it's pretty noticable anyway, well unless colorblind i guess.

**Comment by 00aquir** [comment]
It's not about noticeability, but ease of access on a big-sized list.

**Comment by Super3222** [comment]
This sounds interesting.
